core/src/test/java/apoc/export/arrow/ArrowTest.java

```diff
@@ -60,7 +64,10 @@ public class ArrowTest {
     public static DbmsRule db = new ImpermanentDbmsRule()
             .withSetting(
                     GraphDatabaseSettings.load_csv_file_url_root,
-                    directory.toPath().toAbsolutePath());
+                    directory.toPath().toAbsolutePath())
+            .withSetting(
+                    GraphDatabaseInternalSettings.default_cypher_version,
```

Comment:
This setting is being replaced by `GraphDatabaseSettings.default_language`. Unfortunately that is not hooked up yet ([PR](https://github.com/neo-technology/neo4j/pull/29353)), so go ahead and merge this and I sort it out later.